client/www/components/dash/explorer/EditNamespaceDialog.tsx

```diff
@@ -1563,6 +1553,10 @@ function EditAttrForm({
   }
 
   async function deleteAttr() {
+    // update the recently deleted attr cache
+    setTimeout(() => {
```

Comment:
Nit: would move this below the `await` on pushOps, so you are guaranteed that this will fetch once reactor has pushed

Reply:
its not guaranteed since pushOps doesn't wait for a response confirming its been deleted, which is why i added the setTimeout but i moved it to be after to at least read better.

Reply:
I _think_ if the user is online `pushOps` will await until we get a `transact-ok`, which effectively means the db has confirmed the delete 

https://github.com/instantdb/instant/blob/main/client/packages/core/src/Reactor.js#L582